test/ecdsatest.c

```diff
@@ -57,343 +50,269 @@ static int restore_rand(void)
     return 1;
 }
 
-static int fbytes_counter = 0, use_fake = 0;
-static const char *numbers[8] = {
-    "651056770906015076056810763456358567190100156695615665659",
-    "6140507067065001063065065565667405560006161556565665656654",
-    "8763001015071075675010661307616710783570106710677817767166"
-        "71676178726717",
-    "7000000175690566466555057817571571075705015757757057795755"
-        "55657156756655",
-    "1275552191113212300012030439187146164646146646466749494799",
-    "1542725565216523985789236956265265265235675811949404040041",
-    "1456427555219115346513212300075341203043918714616464614664"
-        "64667494947990",
-    "1712787255652165239672857892369562652652652356758119494040"
-        "40041670216363"
-};
+static int use_fake = 0;
+static const char *numbers[2];
 
 static int fbytes(unsigned char *buf, int num)
 {
     int ret = 0;
+    static int fbytes_counter = 0;
     BIGNUM *tmp = NULL;
 
     if (use_fake == 0)
         return old_rand->bytes(buf, num);
 
     use_fake = 0;
 
-    if (fbytes_counter >= 8)
+    if (fbytes_counter >= 2)
```

Comment:
Probably better to use `OSSL_NELEM()` instead of `2` ?

Reply:
Thanks! fixed in c51ffb56485860f631540b5b329397eefff23bf6